src/test/java/seedu/planner/logic/commands/SetUpCommandTest.java

```diff
@@ -30,19 +32,20 @@ public void execute_validArgs_success() {
 
         String expectedMessage = String.format(MESSAGE_SET_UP_SUCCESS, VALID_MAJOR_CS, VALID_FOCUS_AREA_SE);
         Model expectedModel = new ModelManager();
-        expectedModel.setUpUserProfile(VALID_MAJOR_CS, Set.of(VALID_FOCUS_AREA_SE));
+        expectedModel.setUpUserProfile(Major.COMPUTER_SCIENCE, Set.of(FocusArea.SOFTWARE_ENGINEERING));
```

Comment:
I think it’s good to standardise on whether the named constant is used here

Reply:
I changed everything behind `ModelManager` to use stronger type (`Major` and `FocusArea` enums) rather than strings so that they don't have to worry about invalid values. Error checking is done at `SetupCommand`.